src/atfile.c

```diff
@@ -89,7 +89,7 @@ CAMLprim value caml_extunix_fstatat(value v_dirfd, value v_name, value v_flags)
 CAMLprim value caml_extunix_unlinkat(value v_dirfd, value v_name, value v_flags)
 {
   CAMLparam3(v_dirfd, v_name, v_flags);
-  int dirfd = Int_val(dirfd);
+  int dirfd = Int_val(v_dirfd);
```

Comment:
omg %)

Reply:
Yes, I know -_-
yay for compiler warnings